core/src/main/java/tc/oc/pgm/kits/KitParser.java

```diff
@@ -435,6 +441,24 @@ private List<Color> parseColors(List<Node> nodes) throws InvalidXMLException {
     return colors;
   }
 
+  public ItemStack parseBanner(Element el) throws InvalidXMLException {
+    ItemStack itemStack = parseItem(el, Materials.BANNER);
+    BannerMeta meta = (BannerMeta) itemStack.getItemMeta();
+
+    DyeColor color = XMLUtils.parseDyeColor(XMLUtils.getRequiredAttribute(el, "base-color"));
+    COLOR_UTILS.setColor(itemStack, color);
```

Comment:
```suggestion
    DyeColor color = XMLUtils.parseDyeColor(XMLUtils.getRequiredAttribute(el, "base-color"));
    COLOR_UTILS.setColor(itemStack, color);
    
    BannerMeta meta = (BannerMeta) itemStack.getItemMeta();
```
unsure if this makes a difference, but i believe in creating meta after any previous change is done to avoid things being lost (eg: if setColor was modifying the item meta, those changes would be lost when applying the other meta that was created before calling setColor)